### PR TITLE
Many small fixes for easier management and rapid building

### PR DESCRIPTION
**Pull request by drivera-armedia**
Many small, menial modifications that will make it easier for rapid-rebuilds during testing. Added a mechanism to the startup script to properly set/restore ownership on the data directory, which can be a problem sometimes.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,56 +3,85 @@
...
-        MAINTAINER="Armedia LLC"
+#
+# Basic Parameters
+#
+ARG ARCH="amd64"
+ARG OS="linux"
+ARG VER="5.16.2"
+ARG JMX_VER="0.17.0"
+ARG PKG="activemq"
+ARG AMQ_USER="${PKG}"
```
Comment (david-oc-miller) [inline Dockerfile]: The other ARGs are all generic (no reference to AMQ), but this ARG is "AMQ_USER", do you anticipate some containers having more than one user?  Should this be APP_USER or something?

File: Dockerfile [hunk 2]
```diff
@@ -3,56 +3,85 @@
...
-USER activemq
-ENTRYPOINT ["/app/startup.sh"]
+COPY jmx-prometheus-config.yaml startup.sh ./
+
+#
+# Final file organization
+#
+RUN ln -s "/app/${ACTIVEMQ}" "/app/${PKG}" \
+    && cd "${PKG}" \
+    && rm bin/activemq-diag bin/env bin/wrapper.jar activemq-all-5.16.2.jar conf/*.ts conf/*.ks \
```
Comment (david-oc-miller) [inline Dockerfile]: The version 5.16.2 is hardcoded in this command, is it OK, or replace with $VER?

**Review by david-oc-miller — DISMISSED** [review]

File: Dockerfile (hunk 1 above)
Comment (drivera-armedia) [reply Dockerfile]: Plz revew latest changes

File: Dockerfile (hunk 2 above)
Comment (drivera-armedia) [reply Dockerfile]: Plz revew latest changes

**Review by david-oc-miller — APPROVED** [review]

**Pull request merged**